### PR TITLE
To allow path information when doing a singelton lastsnap picture v2

### PR DESCRIPTION
**Pull request by jdodgen**

### CODE REVIEW

File: event.c [hunk 1]
```diff
@@ -349,12 +349,17 @@ static void event_image_snapshot(struct context *cnt, int type ATTRIBUTE_UNUSED,
...
             void *dummy2 ATTRIBUTE_UNUSED, struct tm *currenttime_tm)
 {
-    char fullfilename[PATH_MAX];
-
-    if (strcmp(cnt->conf.snappath, "lastsnap")) {
-        char filename[PATH_MAX];
-        char filepath[PATH_MAX];
-        char linkpath[PATH_MAX];
+	char fullfilename[PATH_MAX];
+	char filename[PATH_MAX];
```
Comment (tosiara) [inline event.c]: And still whitespace change detected

File: event.c [hunk 2]
```diff
@@ -383,7 +388,9 @@ static void event_image_snapshot(struct context *cnt, int type ATTRIBUTE_UNUSED,
             return;
         }
     } else {
-        snprintf(fullfilename, PATH_MAX, "%s/lastsnap.%s", cnt->conf.filepath, imageext(cnt));
+		mystrftime(cnt, filepath, sizeof(filepath), cnt->conf.snappath, currenttime_tm, NULL, 0);
+		snprintf(filename, PATH_MAX, "%s.%s", filepath, imageext(cnt));
+		snprintf(fullfilename, PATH_MAX, "%s/%s", cnt->conf.filepath, filename);
```
Comment (tosiara) [inline event.c]: Indention looks not in tact

**Comment by tosiara** [comment]
By the way, you don't need to delete pull and recreate it if you found something wrong or need to change. Just make additional commit is the same branch and push to github. It will be automatically added to the pull request

**Comment by jdodgen** [comment]
Ok thanks still learning git.

I find the indentation to be odd, some of the existing code seems to have 4
spaces inplace of a tab.  Is the preference for tabs or spaces.

*Jim Dodgen*

**Comment by jdodgen** [comment]
After further analysis I think the standard is spaces.

The event.c tabs were all mine. and have been corrected.

in config.c there are a few places other than min that have tabs,  is it ok
if I fix these also?


*Jim Dodgen*

**Comment by Mr-Dave** [comment]
Is this resolving a bug in how it was saving files?  If so, do you have a use case?  The documentation indicates that all the files are supposed to be saved in a directory relative to target_dir.  Is this so the snapshots can be saved in a subdirectory under target_dir?

**Comment by jdodgen** [comment]
I'm away from my development system but as I remember the change was two
fold.  If you added directory path to the name then it failed the test for
the singleton "snapshot".  In addition I use it to force a path to
/dev/shm.  I don't think it breaks anything existing.  It might even be
considered a bug fix.

*Jim Dodgen*

**Comment by jdodgen** [comment]
I must add it is still a relative position to the target directory i just
use a collection of  ../../../ things to get to my shared memory.

*Jim Dodgen*

**Pull request merged**